_sections/videocalling-android-studio.md

```diff
@@ -21,13 +21,13 @@ You can also call any collaborator through the “Search Everywhere” list. You
 
 #### Step 2
 
- You will then get a pop-up notification that you have sent the request to call and are waiting for your teammate to accept.
+ You will then get a pop-up notification that you have sent the request to call and are waiting for your teammate to accept. They then have fifteen seconds to pick up, otherwise, they will receive a notification of a missed call.
```

Comment:
The call gets automatically canceled after 15 seconds, leaving your teammate a notification about the missed call with the option to call back.